controllers/networking-calico/charts/internal/calico/templates/rbac.yaml

```diff
@@ -208,89 +210,3 @@ subjects:
 - kind: ServiceAccount
   name: calico-kube-controllers
   namespace: kube-system
----
-apiVersion: {{ include "rbacversion" . }}
```

Comment:
Where did this cluster role go?

Reply:
here https://github.com/gardener/gardener-extensions/pull/432/files#diff-c202b3b53bd963173804776737230878R28

Reply:
Thanks!